src/components/Autosuggest/Autosuggest.scss

```diff
@@ -66,9 +66,10 @@
     }
 
     &__list--focused {
-        max-height: 200px;
+        max-height: 215px;
```

Comment:
This hard-coded `max-height` number breaks the idea of `listRenderer` property

Reply:
I'm not really sure how it relates to `listRenderer`, that one is responsible for each item in the list, while this one is for the max-height of the dropdown. The dropdown is scrollable, and hence the items will show up regardless of their height.

Reply:
but we can move it to a variable if that is what you meant

Reply:
I mean, that `215px` is not just some random value, it plays perfectly with the current height of the item, which is `43px` (`43 * 5 = 215`).
And `listRenderer` allows to render any external component as an item. So it would be logical that max-height is calculated based on Item's height.

Reply:
perhaps when we have a usecase for it, we can add a function for figuring it out.

Reply:
then let's also remove `listRenderer` property, cause now it's hafly done then.

Reply:
we use it to render the poweredByGoogle img in an absolute position within the dropdown. It is fully done. 
BTW: there was no requirement to adjust the height to show 5 items. If the items are very short (in height) that would not look good...